src/main/java/edu/kis/vh/nursery/Stack.java

```diff
@@ -0,0 +1,9 @@
+package edu.kis.vh.nursery;
+
+public interface Stack {
```

Comment:
Names of operations do not correspond to operating the stack, thus do not reflect the interface name.